### PR TITLE
Fixes app crash on deleting downloaded photos

### PR DESCRIPTION
**Pull request by anantprsd5**
Fixes App crashing issue which occurred while trying to delete downloaded photos

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/Settings.java [hunk 1]
```diff
@@ -267,8 +268,15 @@ public void onClick(DialogInterface dialog, int which)
 										}*/
 										
 									}
-									getActivity().sendBroadcast(new Intent(Intent.ACTION_MEDIA_MOUNTED, Uri.parse("file://"
-								            + Environment.getExternalStorageDirectory()))); 
+                                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Can you pls properly items the code :)

Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: There are shortcuts in IDE which you can use for this. 
My suggestion is to use shortcut on every file you work :)

Comment (anantprsd5) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Okay, will make the changes soon.

**Comment by anantprsd5** [comment]
@Abhi2424shek @pa1pal I have made the required changes. Please have a look at this.

**Pull request merged**